song-server/src/main/java/org/icgc/dcc/song/server/config/SecurityConfig.java

```diff
@@ -34,7 +34,7 @@
 @EnableResourceServer
 public class SecurityConfig extends ResourceServerConfigurerAdapter {
 
-  @Value("${auth.server.uploadScope}")
+  @Value("${auth.server.suffix}")
   private String uploadScope;
 
```

Comment:
The data probably wouldn't need to be segregated either given our unique ID generation.  You could have different SONG microservices talking to a single persistence backend but unable to touch each others data due to the `suffix` being injected at startup.